src/lib/drivers/device/nuttx/SPI.cpp

```diff
@@ -221,6 +226,7 @@ SPI::_transferhword(uint16_t *send, uint16_t *recv, unsigned len)
 	SPI_SETMODE(_dev, _mode);
 	SPI_SETBITS(_dev, 16);			/* 16 bit transfer */
 	SPI_SELECT(_dev, _device, true);
+	SPI_CS_TO_SCK_DELAY(_dev, _cs_to_sck_ns);
```

Comment:
```suggestion
	px4_udelay(_cs_to_sck_us);
```

Store the delay in microseconds.